brownie/project/main.py

```diff
@@ -958,3 +959,21 @@ def _stream_download(download_url: str, target_path: str) -> None:
 
     with zipfile.ZipFile(BytesIO(content)) as zf:
         zf.extractall(target_path)
+
+
+def _get_mix_default_branch(mix_name: str) -> str:
+    """Get the default branch for a brownie-mix repository.
+
+    Arguments
+    ---------
+    mix_name : str
+        Name of a brownie-mix repository without -mix appended.
+
+    Returns
+    -------
+    str
+        The default branch name on github.
+    """
+    REPO_GH_API = "https://api.github.com/repos/brownie-mix/{}-mix".format(mix_name)
+    r = requests.get(REPO_GH_API).json()
```

Comment:
We use custom headers for requests made with brownie, they are available via `REQUEST_HEADERS` and should be added to this request (also see other requests in this file).

You should also check if the request response is valid and contains what you need and throw with a more meaningful error otherwise. We generally check `status_code` and if the required data field exists. You can look at other requests in the file to get an Idea.

Reply:
@matnad resolved now with cfc73a1. Thanks for the pointer!